doc/src/manual/interfaces.md

```diff
@@ -591,3 +593,27 @@ yields another `SparseVecStyle`, that its combination with a 2-dimensional array
 yields a `SparseMatStyle`, and anything of higher dimensionality falls back to the dense arbitrary-dimensional framework.
 These rules allow broadcasting to keep the sparse representation for operations that result
 in one or two dimensional outputs, but produce an `Array` for any other dimensionality.
+
+### [Extending `broadcast!`](@id extending-in-place-broadcast)
+Extending `broadcast!` (in-place broadcast) should be done with care, as it is easy to introduce
```

Comment:
I'm not sure, does there need to be a space after the section heading?

Reply:
You're right.